TODO.md

```diff
@@ -15,3 +15,17 @@
 - Combine rolls + rerolls?
 - Non-removal Kickoff result values?
 - Include expected value of armor/injury rolls in rolls that could result in KDs?
+
+
+------------------------------------------------------------------------------------
+
+1. Rework base values for ball handling and hitting (and make sure they are including probabilities as param), make sure they have common denominator
```

Comment:
So, I think this shows that my fundamental theory about how to model the value of players/actions isn't showing through, so I'll try to lay it out here.

Here's the thought process that I'm basic action values on:
1. Players are valuable for their ability to take actions.
2. TV for each player roughly represents the effectiveness of that players actions.
3. The result of most dice rolls is to either remove an opponent players ability to do actions, move the ball, or (on failure) remove your own players ability to do actions (either by removing a player for some number of turns, or by a turnover, which basically stops the rest of your team from taking actions for the rest of your turn).
4. Because of 1-3, all the actions values are based on the fraction of TV that a player is worth in respect to either the other players on the pitch, or your whole team.
5. There will be a scaling modifier based on the distance of a player to the ball (to represent ball control value) and distance to the end of the field (to represent scoring potential). These will modify player TVs.

The thing I like about this formulation is that it implicitly represents advice like "Do safe moves first", and "Prioritize the ball", which should make the tool useful as a learning tool, and not just a dice-griping tool.